miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleClassService.java

```diff
@@ -113,4 +114,25 @@ public SampleClass inferStockFromAliquot(SampleClass sampleClass) {
     return stockClass;
   }
 
+  @Override
+  public SampleClass inferTissueFromStock(SampleClass sampleClass) {
```

Comment:
That might be true now, but I don't know that it will be true in general. Given there are separate, unambiguous tissues, I don't see the harm in this.